pkg/api/v1/routes/routes.go

```diff
@@ -139,10 +139,10 @@ func (r *Routes) Routes(g *gin.RouterGroup) {
 	if r.enableServerEnroll {
 		serverEnroll := g.Group("/serverEnroll")
 		serverEnroll.POST("/:uuid", r.composeAuthHandler(createScopes("server")), wrapAPICall(r.serverEnroll))
-		serverDelete := g.Group("/serverDelete")
-		serverDelete.DELETE("/:uuid", r.composeAuthHandler(createScopes("server")), wrapAPICall(r.serverDelete))
 		// Create a new server ID when uuid is not provided.
 		serverEnroll.POST("/", r.composeAuthHandler(createScopes("server")), wrapAPICall(r.serverEnroll))
+		serverDelete := g.Group("/server")
```

Comment:
we should use the servers group at line 148 for this. I think you can just move that line above this block. Then line 144 is not needed and line 145 becomes: `servers.DELETE("", r.composeAuthHandler...`